target/linux/ramips/dts/U25AWF.dts

```diff
@@ -0,0 +1,114 @@
+/dts-v1/;
+
+/include/"mt7620n.dtsi"
+
+/{
+	compatible="u25awf","ralink,mt7620n-soc";
+	model="KimaxU-25AWF";
+
+	palmbus@10000000{
+		gpio0:gpio@600{
+			status="okay";
```

Comment:
please use whitespaces around the equal sign. Without the equal signs it is kinda hard to read.

The same applies to all ```{```. They should have a leading whitespace.

The same applies to all ```:```. They should have a trainling whitespace.

Have a look how it is done in the other dts files.